pkg/model/config/config.go

```diff
@@ -87,10 +70,10 @@ func (config Config) HasResource(target GVK) bool {
 
 // AddResource appends the provided resource to the tracked ones
 // It returns if the configuration was modified
-// NOTE: this works only for v2, since in v1 resources are not tracked
+// NOTE: in v1 resources are not tracked, so we return false
 func (config *Config) AddResource(gvk GVK) bool {
 	// Short-circuit v1
-	if config.Version == Version1 {
+	if config.IsV1() {
 		return false
 	}
```

Comment:
```suggestion
```
I do not think that it should be added here as well. The V1/V2 checks are the responsibility of the command impl. If for the command A addResources just can be done when isV1 then this validation should be made in the command as itt was done before.

Reply:
This is needed, if we do not add it here v1 projects will add tracked resources and they didn't. I removed every check that was not needed but this is needed. Not having it completely breaks v1.

Reply:
If we do this, v1 configuration file will track resources. It will break backwards compatibility.